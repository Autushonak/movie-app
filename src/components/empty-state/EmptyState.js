import styled from 'styled-components';
import { colors } from '../../constants/colors';

const EmptyState = styled.h1`
  color: ${colors.WHITE};
`;
export default EmptyState;
