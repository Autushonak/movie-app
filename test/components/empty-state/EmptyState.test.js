import 'jest-styled-components';
import React from 'react';
import { shallow } from 'enzyme';
import EmptyState from '../../../src/components/empty-state/EmptyState';

describe('EmptyState ', () => {
  test('should be rendered correctly', () => {
    const component = shallow(<EmptyState>Empty state text</EmptyState>);
    expect(component).toMatchSnapshot();
  });
});
