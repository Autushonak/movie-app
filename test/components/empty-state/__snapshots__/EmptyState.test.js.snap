// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyState  should be rendered correctly 1`] = `
.c0 {
  color: #FFFFFF;
}

<h1
  className="c0"
>
  Empty state text
</h1>
`;
